docs/ResourceInterface.md

```diff
@@ -24,105 +26,76 @@ public function getId(): int
+
+### relationships()
+Returns the resource relationships.
 
-Example of resource without relationships:
 ```php
 public static function relationships(): RelationshipsCollection
 {
     return new RelationshipsCollection();
 }
 ```
 
-There are 2 types of relationships `To-One` and `To-Many` relationships.
-
-### To-One
-To define new "To-One" relationship we must add new [Sowl\JsonApi\Relationships\ToOneRelationship](../src/Relationships/ToOneRelationship.php).
-
-To-One relationship returns single resource in the responses.
-
-2 required parameters must be provider and 1 optional.
-
-#### name
-First parameter must be the name of relationships that will be used for generating building endpoints.
-For example `GET /user/1/{name}` or `GET /user/1/relationships/{name}`
+This method must return an instance of
+[`Sowl\JsonApi\Relationships\RelationshipsCollection`](src/Relationships/RelationshipsCollection.php).
 
-#### class
-The resource class of the relation that also must implement `ResourceInterface`.
+Two types of relationships exist, `To-One` and `To-Many`.
 
-#### property
-Optional parameter that is defining the entity property name, by default the `name` param value will be used.
+#### To-One Relationship
+Returns a single resource in the responses.
 
 ```php
+// User Entity
 public static function relationships(): RelationshipsCollection
 {
     return new RelationshipsCollection([
-        ToOneRelationship::create('user', User::class)
+        ToOneRelationship::create('country', Country::class)
     ]);
 }
 ```
 
-### To-Many
-To define new "To-Many" relationship we must add new [Sowl\JsonApi\Relationships\ToManyRelationship](../src/Relationships/ToManyRelationship.php).
+The `ToOneRelationship::create()` method accepts 3 parameters:
 
-To-Many relationship returns list of resources in the responses.
+- `name` (required):
 
-3 required parameters must be provider and 1 optional.
+Name of the relationship that will be used in the endpoint:
 
-#### name
-First parameter must be the name of relationships that will be used for generating building endpoints.
-For example `GET /user/1/{name}` or `GET /user/1/relationships/{name}`
+i.e: `GET /user/1/{name}` or `GET /user/1/relationships/{name}`
 
-#### class
-Resource class of the relation that also must implement `ResourceInterface`.
+i.e: `GET /user/1/country` or `GET /user/1/relationships/country`
 
-#### mappedBy
-Property on the resource class of the relationship that represents invert relation to the parent resource.
+- `class` (required):
 
-This param can be used for generating queries for fetching list of resources.
+The entity (resource) class being the relation.
 
-```php
-->innerJoin("user.$mappedBy", 'relation')
-->where("relation.users = ${resource->getId()}")
-```
+- `property` (optional):
 
-#### property
-Optional parameter that is defining the entity property name, by default the `name` param value will be used.
+Override the default value of `{name}` for custom endpoints.
```

Comment:
It must be improved

It's can be something like:

"Override the relationship property name that will be the `{name}` by default."
for example if you have `protected Country $userCountry` but you want to have simple "country" relationship name, you need to provide "userCountry" as a value.

Reply:
Great feedback. Thxs